README.md

```diff
@@ -1,2 +1,58 @@
-# comp110-worksheet-5
-Base repository for COMP110 worksheet 5
+# James Vanstone Comp110 Worksheet 5
+
+## Computational Complexity
+
+An implementation in Python to aide in code understanding.
+def HASDUPLICATE(alist):
+	n = len(alist)
+	for i in range(0, n-1):
```

Comment:
This should be `range(0, n)` (in Python the upper bound is not included in the range)